Marlin/src/pins/mega/pins_WANHAO_ONEPLUS.h

```diff
@@ -92,20 +92,16 @@
 #define KILL_PIN           64
 
 //
-// LCD / Controller
-//
-
-#if HAS_SPI_LCD
```

Comment:
These pins are only needed if there is an "SPI LCD" and this is the usual wrapper for these pins. The Wanhao i3 Plus, for example, uses the `MKS_MINI_12864` which produces these defines:

`MKS_MINI_12864` => `MINIPANEL` => `IS_ULTIPANEL` => `ULTIPANEL` `IS_ULTRA_LCD` => `ULTRA_LCD` => `HAS_SPI_LCD`